pkgs/by-name/mp/mpdris2-rs/package.nix

```diff
@@ -0,0 +1,31 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "mpdris2-rs";
+  version = "0.2.3";
+
+  src = fetchFromGitHub {
+    owner = "szclsya";
+    repo = "mpdris2-rs";
+    rev = version;
+    hash = "sha256-0m1nRanzqzhu6dKDvHFX4srfBTLyh5ln6sVBWBGxxw0=";
+  };
+
+  cargoHash = "sha256-WYk9vXhgAepitO3Smv/zxYt+V+aiB786cCfOcaGruQQ=";
+
+  postInstall = ''
+    install -Dm644 misc/mpdris2-rs.service -t $out/share/systemd/user
```

Comment:
This file contains `ExecStart=/usr/local/bin/mpdris2-rs`, which will not work.

Reply:
Don't remember how I used this but I no longer use mpdris2-rs and recommend https://github.com/eonpatapon/mpDris2 instead. Thanks for reviewing!